packages/zend-rest/composer.json

```diff
@@ -21,6 +23,9 @@
             "Zend_Rest": "library/"
         }
     },
+    "suggest": {
+        "zf1s/zend-config": "Used in special situations or with special adapters"
```

Comment:
`Zend_Config` is not used at all - can be removed from `suggest`. Type hinting an instance of `Zend_Config` in `Zend_Rest_Route::getInstance()` does not make the class/package itself required.

Reply:
Just so I understand: the type hinting, if not installed, won't lead to an error?
Or you mean type hinting-only use is not enough to make it required?

Reply:
Both are correct